x-pack/plugin/monitoring/src/test/java/org/elasticsearch/xpack/monitoring/test/MonitoringIntegTestCase.java

```diff
@@ -124,10 +124,6 @@ protected void deleteMonitoringIndices() {
         assertAcked(client().admin().indices().prepareDelete(ALL_MONITORING_INDICES));
     }
 
-    protected void ensureMonitoringIndicesYellow() {
-        ensureYellowAndNoInitializingShards(".monitoring-es-*");
```

Comment:
`ALL_MONITORING_INDICES` is `"monitoring-*` and this is `.monitoring-es-*`. I doubt the difference would actual matter. But probably still better to be safe and use `.monitoring-es-*` in awaitClusterState?

Reply:
Ah well spotted :) Yeah it doesn't matter I think, we're only monitoring ES itself in this test, but I fixed that up.